content/docs/alerting/configuration.md

```diff
@@ -408,6 +408,30 @@ api_key: <string>
 # Comma separated list of tags attached to the notifications.
 [ tags: <tmpl_string> ]
 ```
+## `<victor_ops_config>`
+
+VictorOps notifications are sent out via the [VictorOps API](https://portal.victorops.com/public/api-docs.html#/)
```

Comment:
That's not the API we're using: https://github.com/prometheus/alertmanager/blob/master/config/config.go#L284